### PR TITLE
New task steps config

### PR DESCRIPTION
**Pull request by Purg**
Revision to the task steps configuration format for the new HMM direction for a future revised task monitor node. This is intended to replace the `mit_ll_eval_one_coffee_recipe_steps.json` when the time comes.

### CODE REVIEW

**Comment by Purg** [comment]
@alexander-lynch Matt suggested that you complete this PR, filling in the part marked "TODO".

**Comment by mattbrown11** [comment]
LGTM

**Comment by Purg** [comment]
I'm adding a couple comments to describe fields in the config.

File: config/tasks/task_steps_config-recipe_coffee.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    activity_id: 25
+    description: >-
+      Remove the coffee grounds and paper filter from the dripper
+  - id: 25
+    activity_id: 26
+    description: >-
+      Discard the coffee grounds and paper filter
+
+hmm:
+  num_steps_can_jump_fwd: 1
```
Comment (mattbrown11) [inline config/tasks/task_steps_config-recipe_coffee.yaml:150]: Can we add
dt : Time (seconds) between time steps of HMM.

**Comment by mattbrown11** [comment]
We don't have a reference to a background class. We have a few options here.
1) We can enforce that the first step of every recipe is 'background', and then my HMM class can parse this yaml and infer that the first step (and the associated activity class enum) is a background activity. This way, my HMM knows which activity enum to treat as a background action.
2) We explicitly do not include a background step in the recipe. But internally, my HMM needs to assign a background action, so it needs to know where to find the confidence for that action in the classifier confidences. Background could always be index 0 or always called 'background' if we are going with string-based encoding.

**Comment by Purg** [comment]
> We don't have a reference to a background class. ...

Your HMM parameterization could have a parameter like `activity_background_class_id` that we set to an integer, e.g. `0`.

**Pull request merged**